src/test/shell/bazel/android/android_integration_test.sh

```diff
@@ -464,4 +464,31 @@ EOF
   fi
 }
 
+function test_lots_of_assets {
+  write_hello_android_files
+  setup_android_sdk_support
+
+  cat > java/com/example/hello/BUILD <<'EOF'
+LONG_PATH = "a" * 200
+RANGE = [str(i) for i in range(5000)]
```

Comment:
This would create an argument list that's ~1MB in length, right? You mentioned in #13295 that the MacOS argument limit is 256KB, so wouldn't this range exceed that limit still (is the test passing for your MacOS machine)?